### PR TITLE
Drive info

### PR DESCRIPTION
**Pull request by Blacklotus89898**

### CODE REVIEW

File: angular_ui_app/src/app/components/drive/drive.component.ts [hunk 1]
```diff
@@ -1,10 +1,116 @@
...
+
+    // starts the liistener
+    this.wheel_listen();
+    this.position_listen();
+  }
+
+  // functions
+
+  // angles are in radians
+  euler_from_quaternion(x: number, y: number, z: number, w: number) {
```
Comment (amotaouakkil123) [inline angular_ui_app/src/app/components/drive/drive.component.ts:83]: Why is this function necessary? Do we not have an IMU with known units? It seems unnecessary to put it in drive too.

File: angular_ui_app/src/app/app.component.html [hunk 2]
```diff
@@ -1,3 +1,6 @@
 <div style="color: #fff; font-family: 'Courier New', Courier, monospace;">
-    <app-drive-page/>
+    <!-- navbar in header -->
```
Comment (amotaouakkil123) [inline angular_ui_app/src/app/app.component.html:2]: Are we waiting for more in this app part?

File: angular_ui_app/src/app/components/navbar/navbar.component.html [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<div class="navbar" style="flex-wrap: wrap; align-content: space-around; padding: 0 10%;">
```
Comment (amotaouakkil123) [inline angular_ui_app/src/app/components/navbar/navbar.component.html:1]: What is implied by navbar? The rest of the names are understood, but I can't seem to get what navbar means?

**Review by amotaouakkil123 — CHANGES_REQUESTED** [review]

**Comment by JosephSaliba01** [comment]
@Blacklotus89898 Could you explain what this PR is for

File: angular_ui_app/src/app/components/drive/drive.component.ts (hunk 1 above)
Comment (Blacklotus89898) [reply angular_ui_app/src/app/components/drive/drive.component.ts:83]: I think it was previously used in the old UI. There might be chance that it's no longer used.

File: angular_ui_app/src/app/app.component.html (hunk 2 above)
Comment (Blacklotus89898) [reply angular_ui_app/src/app/app.component.html:2]: I think I only left this comment to remind me that the navbar is moved to header.

File: angular_ui_app/src/app/components/navbar/navbar.component.html (hunk 3 above)
Comment (Blacklotus89898) [reply angular_ui_app/src/app/components/navbar/navbar.component.html:1]: Header is only a container at the top of the UI, while the navbar is the component that has all the clickable links to change pages/views. Essentially, the navbar could be on top (inisde header) or on the side.

**Comment by Blacklotus89898** [comment]
> @Blacklotus89898 Could you explain what this PR is for

This was initially used as a draft PR to see how much it diverged from the angular-ui and it has a lot of deprecated features to be deleted. I'm thinking about deleting it (once I do a proper review it). It will have to wait after my last midterm this Monday.

File: angular_ui_app/src/app/components/drive/drive.component.ts (hunk 1 above)
Comment (amotaouakkil123) [reply angular_ui_app/src/app/components/drive/drive.component.ts:83]: We don't need it anymore.

File: angular_ui_app/src/app/app.component.html (hunk 2 above)
Comment (amotaouakkil123) [reply angular_ui_app/src/app/app.component.html:2]: Remove from PR branch if it is just a local reminder.

File: angular_ui_app/src/app/components/navbar/navbar.component.html (hunk 3 above)
Comment (amotaouakkil123) [reply angular_ui_app/src/app/components/navbar/navbar.component.html:1]: Ok, so this PR is still not done as navbar should be a greater part of the UI skeleton right?

**Comment by amotaouakkil123** [comment]
> > @Blacklotus89898 Could you explain what this PR is for
> 
> This was initially used as a draft PR to see how much it diverged from the angular-ui and it has a lot of deprecated features to be deleted. I'm thinking about deleting it (once I do a proper review it). It will have to wait after my last midterm this Monday.

Sounds good, thanks for the update. please make sure you go through a quick task draft up for the meeting on Tuesday. Shouldn't take you more than 20 minutes.

**Comment by Blacklotus89898** [comment]
> > > @Blacklotus89898 Could you explain what this PR is for
> > 
> > 
> > This was initially used as a draft PR to see how much it diverged from the angular-ui and it has a lot of deprecated features to be deleted. I'm thinking about deleting it (once I do a proper review it). It will have to wait after my last midterm this Monday.
> 
> Sounds good, thanks for the update. please make sure you go through a quick task draft up for the meeting on Tuesday. Shouldn't take you more than 20 minutes.

Just to be sure, you want me to like prepare a plan/list of priority of the task to come for UI right?

**Comment by amotaouakkil123** [comment]
> > > > @Blacklotus89898 Could you explain what this PR is for
> > > 
> > > 
> > > This was initially used as a draft PR to see how much it diverged from the angular-ui and it has a lot of deprecated features to be deleted. I'm thinking about deleting it (once I do a proper review it). It will have to wait after my last midterm this Monday.
> > 
> > 
> > Sounds good, thanks for the update. please make sure you go through a quick task draft up for the meeting on Tuesday. Shouldn't take you more than 20 minutes.
> 
> Just to be sure, you want me to like prepare a plan/list of priority of the task to come for UI right?

Yes, so that your intended review and progress from this PR does not go to waste and so that members have some meat to chew on.

**Comment by Blacklotus89898** [comment]
> > > > > @Blacklotus89898 Could you explain what this PR is for
> > > > 
> > > > 
> > > > This was initially used as a draft PR to see how much it diverged from the angular-ui and it has a lot of deprecated features to be deleted. I'm thinking about deleting it (once I do a proper review it). It will have to wait after my last midterm this Monday.
> > > 
> > > 
> > > Sounds good, thanks for the update. please make sure you go through a quick task draft up for the meeting on Tuesday. Shouldn't take you more than 20 minutes.
> > 
> > 
> > Just to be sure, you want me to like prepare a plan/list of priority of the task to come for UI right?
> 
> Yes, so that your intended review and progress from this PR does not go to waste and so that members have some meat to chew on.
Sounds good, thanks

File: angular_ui_app/src/app/components/navbar/navbar.component.html (hunk 3 above)
Comment (Blacklotus89898) [reply angular_ui_app/src/app/components/navbar/navbar.component.html:1]: Yeah, we could say that I'm making the navbar an individual component that can be reused or moved, and it is currently placed inside the header of the UI.

**Comment by JosephSaliba01** [comment]
@Blacklotus89898 Can I close this PR?

**Comment by Blacklotus89898** [comment]
> @Blacklotus89898 Can I close this PR?

Yeah I think it's no longer relevant and I can always reimplement the navbar when needed

**Comment by JosephSaliba01** [comment]
Closing as changes no longer relevant and can be re-implemented easily later.

**Pull request closed without merging**